doc/ecmp/fine_grained_next_hop_hld.md

```diff
@@ -320,6 +327,8 @@ Following orchagents shall be modified. Flow diagrams are captured in a later se
 The overall data flow diagram is captured in Section 3 for all TABLE updates.
 Refer to section 4 for detailed information about redistribution performed during runtime scenarios.
 
+### vnetorch

```

Comment:
@anish-n Since this approach of internally creating `FGNhgEntry` differs from the existing method of configuring the feature through the `FG_NHG`, `FG_NHG_PREFIX`, and `FG_NHG_MEMBER` tables, it would be helpful to add a brief explanation of the rationale behind introducing this new approach.

Reply:
We want to enable Fine Grained ECMP creation directly using VNET_ROUTE_TUNNEL, vs requiring a user to setup FG_NHG/FG_NHG_PREFIX, as this would be redundant config from a user perspective, we program this via a SDN controller so we have a need to simplify config model where possible. If we needed to support such a feature using FG_NHG and FG_NHG_PREFIX it would require complex schema changes with addition of vrf/vnet on the existing schema, which also comes with the need for this schema to be backwards compatible.